GNUmakefile

```diff
@@ -222,7 +222,8 @@ changelogfmt:
 .PHONY: hclfmt
 hclfmt:
 	@echo "--> Formatting HCL"
-	@find . -path ./terraform -prune -o -name 'upstart.nomad' -prune -o \( -name '*.nomad' -o -name '*.hcl' \) -exec hclfmt -w {} +
+	@find . -path ./terraform -prune -o -name 'upstart.nomad' -prune -o \( -name '*.nomad' -o -name '*.hcl' \) -exec \
```

Comment:
Accidental extra change?

Reply:
Bonus change! The `hclfmt` broke because of a merge error on one of the hcl test files, and the command didn't report which hcl file failed to fmt. This makes it tell you which one broke. I can break it out of this pr.

Reply:
oh nice! - Wanna PR this separately so we get it in master sooner?